src/Dialog/Dialog.js

```diff
@@ -398,7 +412,7 @@ class Dialog extends Component {
      */
     children: PropTypes.node,
     /**
-     * The css class name of the root element.
+     * The `className` to add to the root element.
```

Comment:
Adding the overall props description style in Dialog component.
in [Chip](https://github.com/callemall/material-ui/blob/master/src/Chip/Chip.js):
```
    /**
     * CSS `className` of the root element.
     */
    className: PropTypes.node,
```
in [AppBar](https://github.com/callemall/material-ui/blob/master/src/AppBar/AppBar.js):
```
    /**
     * Applied to the app bar's root element.
     */
    className: PropTypes.string,
```
In [Dialog](https://github.com/callemall/material-ui/blob/master/src/Dialog/Dialog.js):
```
     /**
     * The `className` to add to the title's root container element.
     */
    titleClassName: PropTypes.string,
    /**
    /**
     * The `className` to add to the `Overlay` component that is rendered behind the `Dialog`.
     */
    overlayClassName: PropTypes.string,
    /**
     * The `className` to add to the content container.
     */
    contentClassName: PropTypes.string,
```
I see there is no overall props description style in components.